aces/Loris
# [Raisinbread] Removed references to Reliability module

I removed references to the Reliabilty module in the Raisinbread permissions table. I looked for other references to Reliability throughout Raisinbread and other SQL patches and did not find any more that needed to be deleted.

#### Testing instructions 

With a fresh RB install:

1. Use the query `SELECT * FROM permissions;` in your database to see that there are no reliability permissions.


* Resolves #5876
